src/main/scala/temple/utils/StringUtils.scala

```diff
@@ -5,4 +5,10 @@ object StringUtils {
 
   /** Given a string, indent it by a given number of spaces */
   def indent(str: String, length: Int = 2): String = str.replaceAll("^|(?<=\n)", " " * length)
+
+  /** Given a string and a wrapping char, append the char at each end */
+  def wrap(str: String, char: Char = '"'): String = s"$char$str$char"
+
+  /** Given a string that may contain quotes, automatically escape them */
+  def escapeQuotes(str: String): String = str.split('"').mkString("\\\"")
```

Comment:
Firstly, this doesn't escape backslashes. Secondly, surely this has been done before?

Reply:
How do you mean done before? In a library?

Reply:
And how much santisation do we need? Surely if this is an issue here it's an issue in all the other generation code?

Reply:
I mean that there will be a library function for escaping this stuff. And it's more important here than elsewhere, because everything else I've done has been from identifiers parsed from the source, not free-form command-line input, where it's realistic that unusual characters could appear.

However, this here is simply a JSON array, so use an off-the-shelf JSON generator instead of `buildArrayString`

Reply:
Sure I'll do that - Though I don't think the input here would be command-line input - it'd be generated from the parsed source same as the postgres and go gen

Reply:
Not from command-line input, but it contains command-line commands, which frequently contain “special” characters (in the RUN block)

Reply:
Yeah that's true 

Do you think it's worth adding a dependency on a  whole JSON library for this?

Reply:
I do, as it will cover edge cases, as we’ll almost certainly need to generate JSON in the future